packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.tsx

```diff
@@ -8,4 +8,4 @@ export const ColorPickerGridCell: React.StatelessComponent<IColorPickerGridCellP
   IColorPickerGridCellProps,
   IColorPickerGridCellStyleProps,
   IColorPickerGridCellStyles
->(ColorPickerGridCellBase, getStyles, undefined, { scope: 'ColorPickerGridCell' });
+>(ColorPickerGridCellBase, getStyles, undefined, { scope: 'ColorPickerGridCell' }, true);
```

Comment:
If you're doing this, you should probably also update `ColorPickerGridCellBase` to extend `React.PureComponent` or make it into a function component (which would be pretty simple in this case).

Reply:
I do not know the guidance here. In our codebase, almost all components are pure components.

Fabric on the other hand I think wanted to keep its Base Components not be pure because other consumers might be unknowingly mutating their props without changing their references. Like other examples like Icon.base.tsx are also non-pure components. 
@JasonGore who I have talked with before about this.

Reply:
I realize the other person on that thread was you @ecraig12345  :)

Reply:
My point was mainly that the `pure` setting here should probably mirror whether the component itself is pure or not.

As far as whether ColorPickerGridCell(Base) should be pure or not, my *guess* is that the risk of the prop mutation issue is lower with SwatchColorPicker than with some of the other components. The components which have the worst problems with this are ones that allow setting/storing selected state in either a component prop or individual items (SwatchColorPicker only allows setting the selected item through a prop).

Reply:
Sounds good. I changed it to extend a PureComponent. Thanks !